test/e2e/flowaggregator_test.go

```diff
@@ -427,6 +430,16 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// ToExternalFlows tests the case, where Pod is deployed on the source Node and sending traffic to an external IP,
+	// the flow information is exported as IPFIX flow records.
+	t.Run("ToExternalFlows", func(t *testing.T) {
+		clientName, clientIPs, cleanupFunc := createAndWaitForPod(t, data, data.createNginxPodOnNode, "test-client-", controlPlaneNodeName())
+		defer cleanupFunc()
+		// skipping test on v6 clusters, for no test clusters with reliable external connectivity over v6
```

Comment:
"Skipping the test on IPv6 clusters as there are no clusters with reliable IPv6 external connectivity."